tests/population_tests.c

```diff
@@ -69,16 +69,16 @@ void test_population_create()
 	assert(goal_create_latest_versions(&system_definition, &goal) == 0);
 
 	/* generate the population */
-    retval = population_create(100, &population, &system_definition, &goal);
+	retval = population_create(population_size, &population, &system_definition, &goal);
 	if (retval != 0) {
 		printf("\nDidn't create population: error %d\n", retval);
 	}
 
 	/* check that the size is as expected */
-	if (population.size != 100) {
+	if (population.size != population_size) {
 		population_free(&population);
```

Comment:
I'm not sure if memory would be cleaned up after assert, so since the population is dynamically allocated that's why there's a free here.